metabulo/views.py

```diff
@@ -489,3 +489,21 @@ def get_wilcoxon_test(validated_table, zero_method, alternative):
     data = wilcoxon_test(table, zero_method, alternative)
 
     return jsonify(data), 200
+
+
+@csv_bp.route('/csv/<uuid:csv_id>/analyses/anova', methods=['GET'])
+@use_kwargs({
+    'group_column': fields.Str()
+})
+@load_validated_csv_file
+def get_anova_test(validated_table, group_column=None):
+    measurements = validated_table.measurements
+    groups = validated_table.groups
+
+    group = groups.iloc[:, 0] if group_column is None else groups.loc[:, group_column]
+    if group is None:
+        abort(404, 'invalid group column')
```

Comment:
This should throw a `marshmallow.ValidationError`, which will return a properly formatted 400 error.